src/ranking.jl

```diff
@@ -30,15 +30,23 @@ function ordinalrank!(rks::RealArray, x::RealArray, p::IntegerArray)
     return rks
 end
 
+
+"""
+    ordinalrank(x) -> Array
+
+Return the ordinal ranking of a real-valued array. Ordinal ranking,
+aka "1234" ranking, uses the literal sorted order. All items in `x`
```

Comment:
Minor point: mentioning "1234" in the first sentence as for methods below sounds better to me.